File: 10 Summation of Primes/10.rb
def prime?(num)
  return false if num == 1
  2.upto(num ** 0.5) do |i|
    return false if num % i == 0
  end
  true
end

count = 3;
sum = 2;
while count < 2_000_000
  if (prime?(count))
    sum = sum + count;
  end
  count += 2;
end

p sum

File: 10 Summation of Primes/10.js
var primeCheck = function(number) {
  var i = 2;
  while (i < (Math.sqrt(number) + 1)) {
    if (number % i === 0 && number !== 2) {return false;}
    i ++;
  }
  return true;
};

var count = 2;
var sum = 0;
while (count < 2000000){
  if (primeCheck(count)) {
    sum = sum + count;
  }
  count ++;
}

console.log(sum)
